packages/theme/src/theme/_forms.scss

```diff
@@ -6,6 +6,11 @@
 $form-shadow: inset 0 -1px 0 $gallery;
 $form-focus-shadow: inset 0 -2px 0 $scooter;
 
+@mixin checkOutline() {
+	outline: 2px auto -webkit-focus-ring-color;
```

Comment:
What about non-webkit browsers? Could we have a fallback color for them on the line before maybe?

Reply:
we still use autoprefix for compatibility

Reply:
Indeed, that doesn't fix for non webkit browsers :/
I changed it to use `Highlight` color that takes the default highlight color.